azuredevops/internal/service/core/data_team.go

```diff
@@ -61,7 +65,15 @@ func dataTeamRead(d *schema.ResourceData, m interface{}) error {
 	projectID := d.Get("project_id").(string)
 	teamName := d.Get("name").(string)
 
-	team, members, administrators, err := readTeamByName(d, clients, projectID, teamName)
+	var top int
+	data, ok := d.GetOk("top")
+	if ok {
+		top = data.(int)
+	} else {
+		top = 2147483647
+	}
```

Comment:
With default value in the schema,  this code snippet can change to `top := d.Get('top')`